ete4/smartview/renderer/layouts/evocell_layouts.py

```diff
@@ -43,9 +43,9 @@ def set_node_style(self, node):
         
          if node.is_leaf():
             if node.props.get('UCSC'):
-                tooltip = self.get_html(node)
-                if tooltip:
-                    node.props["tooltip"] = tooltip
+                # tooltip = self.get_html(node)
+                # if tooltip:
+                #     node.props["tooltip"] = tooltip
```

Comment:
I think it is better to remove the code that we don't use, rather than leaving it commented out.

Commented code tends to bloat the implementation and it's harder in the future to know if that was a good alternative or what. It's normally better to comment what the possibilities are, or add a function that chooses between alternatives, and make it clear which one we are using and if in the future we could want to use a different one because some circumstance has changed.